apis/server/volume_bridge.go

```diff
@@ -64,6 +64,7 @@ func (s *Server) createVolume(ctx context.Context, rw http.ResponseWriter, req *
 			status[k] = v
 		}
 	}
+	status["size"] = volume.Size()
 	respVolume.Status = status
```

Comment:
I think we have discussed before. When constructing a returned instance like `respVolume`, try to use **prepared everything and then initialize one**, rather than **initialize one instance and then set values for it**.

Reply:
Modified